### ISSUE TITLE
PDF/A-1B and image transparency

### ISSUE THREAD

**Issue by adrian-kuhn** [jsreport/jsreport#1068]
I appreciate the new (beta) functionality to create PDF/A-1B conform PDF. Everything is working great so far. 
I recently detected, that any opacity information on images are ignored, when generating a PDF/A-1B conform PDF.

For example CSS information for an image will not work (tested with jsreport 3.12.0):
`style="opacity: 0.2"`

I think this is related to the rigid standard itself, which doesn't allow any transparent elements. If this is the case, are there any plans to support the more tolerant standard PDF/A-2 as well? I think this standard would allow transparent elements.

Thank you for your answer and your overall excellent work with jsreport!

**Comment by pofider** [comment]
Yes, it's like you say. The code is here if you would want to tweak it.
We will check the PDF/A-1B support later this year hopefully.
